src/FS.Embedded/FS.Embedded.csproj

```diff
@@ -13,6 +13,7 @@
 
   <ItemGroup>
     <None Include="build\netstandard2.0\*" Pack="true" PackagePath="build\netstandard2.0\" />
+    <None Include="buildMultiTargeting\*" Pack="true" PackagePath="buildMultiTargeting\" />
```

Comment:
Why are we multi targeting? Embedded resources only executes in the inner-build.

Reply:
That's true. But I wanted to ensure configurable properties are always available to the csproj and not just in the inner build.

Reply:
I can't really think of a scenario for a user needing the default props from an outer-build, but I'll trust your judgement.

Reply:
It just makes access to these properties consistent.